### ISSUE TITLE
Benchmark results

### ISSUE THREAD

**Issue by aminya** [zandaqo/structurae#17]
It would be nice to have a markdown file showing the benchmark results.

**Comment by zandaqo** [comment]
Yes, indeed, making benchmarks more accessible and useful would be nice, and I hope to get around it somewhere down the line. 

Though, I doubt the usefulness of just putting the results of existing benchmarks in a file. At the moment, the benchmarks serve more as a development tool to watch for regressions, rather than something for an end user to judge the general performance. Most of the structures are too general and have many use cases making it a) hard to tell what cases are more general and benchmarks would be useful; b) extremely easy to engineer the data and use cases for a necessary effect.
